physionet-django/events/tests_views.py

```diff
@@ -125,7 +125,9 @@ def test_edit_event_invalid(self):
                 'allowed_domains': ''
             })
 
-        self.assertEqual(response.status_code, 302)
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'events/event_edit.html')
+        self.assertContains(response, 'Event with this title already exists')
```

Comment:
@tompollard, Do you prefer to remove the text assertion(`assertContains`) from here? I remember Alistair mention on another PR that this might add overhead later as if someone changes the validation message on forms, they would have to fix the test as well.

Reply:
I'm happy either way! I can see there is some benefit in having this test, but it is also fairly strict. It's easy for someone to fix the test if they want to change the text, so it doesn't bother me (but I can understand why it might bother other people!).

Reply:
yup. sounds good to me. It also looks good to me. without the text assertion, in this particular test, its kind of hard to make sure if the `title` was the issue

i will let it be and refactor if @alistairewj prefers this refactored.